build_tools/azure/test_docs.sh

```diff
@@ -8,4 +8,8 @@ elif [[ "$DISTRIB" == "ubuntu" ]]; then
     source $VIRTUALENV/bin/activate
 fi
 
+if [[ "$BUILD_WITH_ICC" == "true" ]]; then
+    source /opt/intel/inteloneapi/setvars.sh
+fi
```

Comment:
Please add a small comment to explain that some env variables are required to use scikit-learn at runtime when built with ICC.